client/src/components/mapping/FhirMappingPanel/FhirResourceTree/index.tsx

```diff
@@ -195,10 +195,22 @@ const FhirResourceTree = ({ onClickCallback }: Props) => {
       .map(buildNodeFromAttribute)
       .filter(Boolean) as TreeNode[];
 
+  const buildSlicedNode = (attribute: Attribute): TreeNode[] => {
+    const children = attribute.slices.map(buildNodeFromAttribute)
+
+    const copy = Attribute.from({ ...attribute, slices: [] });
+    const base = buildNodeFromAttribute(copy);
```

Comment:
Yes, I agree that it's a bit odd
It's because slices are stored as "children" of a default slice.
For instance, I can have an attribute and then slices that are specifications of it.
And what I want to show is something like
![Capture d’écran 2020-04-21 à 10 58 32](https://user-images.githubusercontent.com/19286277/79846798-27ec2100-83bf-11ea-8fc2-94a69c53bb46.png)

Reply:
gotcha! please add a comment to explain